### PR TITLE
Fix fixall title suffix

### PR DESCRIPTION
**Pull request by Shane-XB-Qian**
clear the 'fix all' meaning:
1, actually it is a 'term' which the 'all' meant all safe fixes only.
2, somehow some '(unsafe)' marker e.g F841 maybe would not be added, vs adding 'all safe fixes' suffix to 'fix all' probably would be a more direct meaning title.

### CODE REVIEW

File: pylsp_ruff/plugin.py [hunk 1]
```diff
@@ -405,7 +405,7 @@ def create_fix_all_code_action(
     document: Document,
     settings: PluginSettings,
 ) -> CodeAction:
-    title = "Ruff: Fix All"
+    title = "Ruff: Fix All (all safe fixes)"
     kind = CodeActionKind.SourceFixAll
 
     # No unsafe fixes for 'Fix all', see https://github.com/python-lsp/python-lsp-ruff/issues/55
```
Comment (Shane-XB-Qian) [inline pylsp_ruff/plugin.py:411]: this is supposed settings.unsafe_fixes is default false and only to that,
or if can be switched to true, then the title perhaps should dyn to change.

**Comment by jhossbach** [comment]
Setting the unsafe_fixes to true will not change the behaviour of the "Fix all" action, this is actually part of the specification.

I don't really have an opinion other than we would be the first to call it "Fix all (safe fixes)", but do we _really_ need this? It is pointed out twice in the README, once in the configuration example and at the end.

**Comment by Shane-XB-Qian** [comment]
> do we really need this?

I think yes, to me I am not the first time confused 'fix all' meaning, specifically when I chose that action, but it would may be actually do nothing (no safe fixed there actually).

--
shane.xb.qian

**Comment by Shane-XB-Qian** [comment]
"Fix all (all safe fixes)"

"Fix all (only safe fixes)"

"Fix all (safe fixes)"

you can choose one.

--
shane.xb.qian

**Comment by Shane-XB-Qian** [comment]
@jhossbach please check again if ok to you.

**Comment by Shane-XB-Qian** [comment]
so is it ok to merge now?

1. as for the desc format, seems the lib code was like following:
```python
    """Optional type.

    Optional[X] is equivalent to Union[X, None].
    """
```
there is no blank line before the close `"""`

2. as for the `checks_with_fixall` var, perhaps it was to make it clean to read.

but anyway those should be not a big deal, let me know if still need something to modify, or you can help me refine it as well. thanks.

**Pull request merged**